installer.tpl.sh

```diff
@@ -37,7 +37,11 @@ detect_os ()
           fi
         fi
       elif [ $os = "devuan" ]; then
-        dist=$(echo $(. /etc/os-release && echo $VERSION) | sed 's/^[[:digit:]]\+ (\(.*\))$/\1/')
+        # $ docker run -it dyne/devuan:chimaera grep VERSION= /etc/os-release
```

Comment:
What this comment is intended for?

Reply:
There are codenames with and without the slash symbol.

Reply:
ok.